### ISSUE TITLE
Swift

### ISSUE THREAD

**Issue by AsimNet** [BadChoice/RVCalendarWeekView#15]
Hello,

is there a swift version of this library ?

here what happens when I use it with swift: 

![screen shot 2016-10-26 at 11 49 23 pm](https://cloud.githubusercontent.com/assets/3624441/19744604/dec778ae-9bd6-11e6-8a53-7413a3e49068.png)

**Comment by BadChoice** [comment]
I never used swift so I'm afraid I can't help you with that,
if you find the solution, just do me a PR 

Thanks!

**Comment by BadChoice** [comment]
Did you find the solution?

**Comment by EricBeato** [comment]
The issue is the Masonry shorthand macro, I solved it by modifying the GCC_PREPOCESSOR_DEFINITIONS in the post_install step of my Podfile definition:

    post_install do |installer|
      installer.pods_project.targets.each do |target|
        target.build_configurations.each do |config|
        config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] ||= ['$(inherited)']
        config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << 'MAS_SHORTHAND=1'
        end
      end
    end

**Comment by BadChoice** [comment]
Sounds great!
I'll add it latter today and see if works!

**Comment by BadChoice** [comment]
I added it to the project pod, but I think it should be added to any project using this lib, do you know if there is a way fo tix this in the podscpec itself?

**Comment by EricBeato** [comment]
Yeah sorry, that's what I meant: this solution should be applied on the project using the library. I didn't tried solving it on the podspec. When I have some time i'll look into it .

**Comment by AsimNet** [comment]
Fork it and submit a pull request

**Comment by IOSDeveloperProgrammer** [comment]
@AsimNet  @BadChoice  Hi First, Thank you !  i need example in swift. how to implement it? can you please provide example. @AsimNet  did you find solution ?

**Comment by BadChoice** [comment]
I tried swift and it was total pain, so I sticked in objective-c
